### ISSUE TITLE
1.7.2 js module config fails to load

### ISSUE THREAD

**Issue by wejendorp** [prettier/prettier#2924]
**Prettier 1.7.2**

```
# myConfig.js
module.exports = { semi: true };
```
```
$ prettier --config ./myConfig.js index.js
Error: Invalid configuration file.
Failed to parse "[..]/myConfig.js" as JSON, JS, or YAML.
```

**Expected behavior:**
Should load js module as config.

**Comment by ikatyang** [comment]
steps to reproduce: 

```sh
mkdir test
cd test && yarn --init
yarn add prettier
echo 'module.exports = { \nsemi: true\n };' > myConfig.js
echo 'const x = 1' > index.js
node_modules/.bin/prettier --config myConfig.js index.js
```

I found it works fine with unbundled version, it seems something wrong with bundling for `consmiconfig`'s `require-from-string` module, `module.parent` is missing.

cc @azz @lydell

---

As a temporary workaround, you have to install prettier from github:

```sh
npm install prettier/prettier
```

**Comment by bstream** [comment]
Regression from `1.7.0`

**Comment by suchipi** [comment]
Should we release a 1.7.3 with this fix? (seeing as it's labeled high-pri)

**Comment by lydell** [comment]
@suchipi Yes, that would be very good.

**Comment by azz** [comment]
Just published [`1.7.3`](https://github.com/prettier/prettier/releases/tag/1.7.3)

**Comment by wejendorp** [comment]
Great, thanks for the quick fix! 💪
